core/server/master/src/main/java/alluxio/master/FaultTolerantAlluxioMasterProcess.java

```diff
@@ -225,7 +225,7 @@ public void stop() throws Exception {
   /**
    * @return whether the master is running
    */
-  protected boolean isRunning() {
+  boolean isRunning() {
     return mRunning;
   }
```

Comment:
Following our discussion about naming `get`-ers, should this be named `getRunning()`?

I know the Java-native approach for boolean getters is to use `is/has` instead but yeah just bringing up that point.

Reply:
The discussion around `get`ers revolved around non-trivial `get`ers, ones with some computation inside of them. Given that this function is trivial (just returning the member variable) and that it follows the Java standard (like `isEmpty` for example), I think it's fine.